tests/implementor/conftest.py

```diff
@@ -325,6 +324,55 @@ def train_speed_fault_configuration(train_speed_fault_configuration_data):
     return TrainSpeedFaultConfiguration.create(**train_speed_fault_configuration_data)
 
 
+# ------------- TrackBlockedFaultConfiguration ----------------
+
+
+@pytest.fixture
+def track_blocked_fault_configuration_data(track: Track) -> dict:
+    return {
+        "start_tick": 30,
+        "end_tick": 300,
+        "description": "test TrackBlockedFault",
+        "affected_element_id": track.identifier,
+        "strategy": "regular",
+    }
+
+
+@pytest.fixture
+def track_blocked_fault_configuration(track_blocked_fault_configuration_data):
+    return TrackBlockedFaultConfiguration.create(
+        **track_blocked_fault_configuration_data
+    )
+
+
+# ------------- PlatformBlockedFaultConfiguration ----------------
+
+
+@pytest.fixture
+def platform() -> Platform:
+    return Platform("fault injector platform")
+
+
+@pytest.fixture
+def platform_blocked_fault_configuration_data(platform: Platform) -> dict:
+    return {
+        "start_tick": 20,
+        "end_tick": 200,
+        "description": "test PlatformBlockedFault",
+        "affected_element_id": platform.identifier,
+        "strategy": "regular",
+    }
+
+
+@pytest.fixture
+def platform_blocked_fault_configuration(
```

Comment:
We have some duplication again 🙃 Please use the other track_blocked_fault_configuration or refactor it.